pipelines/README.md

```diff
@@ -0,0 +1,105 @@
+# Integration Test Pipelines
+
+Follow the instructions [here](https://github.com/RedHatInsights/bonfire-tekton?tab=readme-ov-file#add-the-integration-test-scenario-to-your-application) for getting setup with one of the following integration test pipelines and then configure it to use one of the pipelines on this page.  
+
+- [Basic](#basic)
+- [Basic with no IQE](#basic-with-no-iqe)
+- [Frontend Testing Pipeline](#frontend-testing-pipeline) 
+
+## Basic
+
+### Description
+
+TODO: Write description
+
+
+## Basic with no IQE
+
+### Description
+
+TODO: Write description
+
+
+## Frontend Testing Pipeline
+
+### Description
+
+The frontend testing pipeline (fe_testing.yaml) allows app teams to create a custom task at the `.tekton` directory of an application's repository. [Here](https://github.com/RedHatInsights/insights-chrome/tree/master/.tekton/run-tests-tasks.yml) is an example of this custom task.
```

Comment:
This link is currently a 404

Reply:
Yeah waiting on them to merge, sorry, should have mentioned that! https://github.com/RedHatInsights/insights-chrome/pull/3041